CONTRIBUTING.md

```diff
@@ -99,4 +99,4 @@ In order for the course structure to be maintained, localized files need to adhe
 
 1. Localized lesson files should be in a subdirectory of `content` named after the language abbreviation. For example, lessons translated into Spanish should be housed in `content/es`.
 2. Localized asset files should be in a subdirectory of `assets` named after the language abbreviation. For example, assets localized into Spanish should be housed in `assets/es`.
-3. File names for localized files must be identical to their English counterpart. To be clear, *do not translate file names*. The file name is used as the slug for the article and must be identical between languages.
\ No newline at end of file
+3. File names for localized files must be identical to their English counterpart. To be clear, *do not translate file names*. The file name is used as the slug for the article and must be identical between languages.
```

Comment:
Ugh I used the github web UI to make this PR and it added/removed some whitespace. Silly UI. Pardon the noise. If it bothers you I can fix locally.